system-int-test/test/cmr/system_int_test/search/generic_association_test.clj

```diff
@@ -828,6 +828,13 @@
         (is (= (get-in (first (:body response2)) [:generic-association :revision-id])
                (+ 1 (get-in (first (:body response1)) [:generic-association :revision-id]))))
 
+        ;; The first and second associations contain different associated-item because associated-item
+        ;; should always be related to of the items in the list that's associated to a particular concept.
```

Comment:
I think `of` in `related to of the items` is an extra word.

Reply:
It's should be  ```related to one of the items```